src/components/select/tests/SingleSelectConnected.spec.tsx

```diff
@@ -20,7 +20,7 @@ describe('Select', () => {
 
         const id: string = 'list-box-connected';
 
-        const mountSingleSelect = (items: IItemBoxProps[] = [], props: Partial<ISingleSelectProps> = {}) => {
+        const mountSingleSelect = (items: IItemBoxProps[] = [], props: ISingleSelectProps & any = {}) => {

```

Comment:
why do you need to add this here ?

Reply:
I have a problem with 2 interface, `selected` is defined with the button interface and with the ISingleSelectProps. I dont know how to handle this use case. For a simple uts, I think its legit to use any to avoid a ts error